frontend/src/security/SecuredPage.tsx

```diff
@@ -0,0 +1,29 @@
+import axios from "axios";
+import {AppUser} from "./model/AppUser";
+import React, {Dispatch, SetStateAction} from "react";
+import DeleteAccount from "./DeleteAccount";
+import "./css/SecuredPage.css";
+
+type SecuredProps = {
+    fetchUsername: () => void,
+    appUser: AppUser
+    fetchUser: () => void
+    setUsername: Dispatch<SetStateAction<string | undefined>>
+}
+export default function SecuredPage(props: SecuredProps) {
+    function logout() {
+        axios.get("/api/app-users/logout")
+            .then(props.fetchUsername)
+    }
+
+    return <div className={"border"}>
+        <div className={"logged-in"}>
+            <h1>Logged in as {props.appUser.role} </h1>
```

Comment:
Sure that the logged in as... Info is h1?

Reply:
text ist jetzt nicht mehr in h1, sondern in div. Es wird dies in anderem branch refaktorisiert.